frontend/components/surplus/SurplusFlow.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div :class="isStagingEnvironment ? 'SplitLayoutStagingContainer' : 'SplitLayoutContainer'">
+        <SplitLayout :step.sync="step">
```

Comment:
Does the page switching with the panel works already?

Reply:
`SplitLayout` is the layout that shows two different layouts/pages at the same time. Clicking a link in the surplus table should open a panel from the right sight. Currently it doesn't work and is not testable. Since this PR will be the last one to be merged, we should wait for both:
- table with surplus auctions
- surplus auction "page"
to be completed and merged to continue working on this PR